core/src/yamlscript/compiler.clj

```diff
@@ -25,6 +27,35 @@
    [yamlscript.debug :refer [www #_xxx]])
   (:refer-clojure :exclude [compile]))
 
```

Comment:
One needs to use `emit-source` and `program-template` to get the correct C++ code